js/views/conversation_view.js

```diff
@@ -529,6 +530,22 @@
             }
         },
 
+        scrollToMessage: function(providedOptions) {
+            const options = providedOptions || {};
```

Comment:
Done. Some of the eslintified files in this PR still have the `provideXXX` pattern, in some cases because mutation of the parameter happens in the function.